src/com/googlecode/shavenmaven/Resolver.java

```diff
@@ -47,17 +49,17 @@ public Resolver(File directory) {
         this(directory, System.out);
     }
 
-    public boolean resolve(Artifact artifact) throws Exception {
```

Comment:
Not sure about this, if you don't like the boolean what about an Enum or do you pass a custom message somewhere?